### PR TITLE
Change STAT -> STAPI

### PR DESCRIPTION
**Pull request by Geosynopsis**

### CODE REVIEW

**Review by m-mohr — CHANGES_REQUESTED** [review]
As the conformance classes change, the examples should also be updated.

File: examples/OpportunityCollection.json [hunk 1]
```diff
@@ -89,6 +89,6 @@
         "title": "Next page of Opportunities",
         "method": "GET",
         "type": "application/geo+json",
-        "href": "stat-api.example.com?page=2"
+        "href": "stapi.example.com?page=2"
```
Comment (m-mohr) [inline examples/OpportunityCollection.json:92]:
```suggestion
        "href": "https://stapi.example.com?page=2"
```

File: examples/ProductCollection.json [hunk 2]
```diff
@@ -151,6 +151,6 @@
         "title": "Next page of Products",
         "method": "GET",
         "type": "application/geo+json",
-        "href": "stat-api.example.com?page=2"
+        "href": "stapi.example.com?page=2"
```
Comment (m-mohr) [inline examples/ProductCollection.json:154]:
```suggestion
        "href": "https://stapi.example.com?page=2"
```

File: extensions/order-status/README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # Overview
 
-- **Conformance URI:** <https://stat-api.example.com/v0.1.0/order-status>
+This document explains the structure of a STAPI **Order Status** request. Operation returns a list/array of statuses that have affected an order.  The most recent status code is the current status of the order.
```
Comment (m-mohr) [inline extensions/order-status/README.md:3]:
```suggestion
```

File: product/README.md [hunk 4]
```diff
@@ -1,8 +1,16 @@
...
-This document explains the structure of a STAT Product.
+This document explains the structure of a STAPI Product.
+
+STAPI Product objects are represented in JSON format and are very flexible. Any JSON object that contains all the required fields is a valid STAPI Product. A Product object contains a minimal set of required properties to be valid and can be extended through the use of constraints and parameters.
+
+## Product Collection Spec
+
+| Element       | Type    | Description |
+| ---- | --- | --- |
+| products | [[Provider](#product-spec)] | List of `Product` offered in the application. |
```
Comment (m-mohr) [inline product/README.md:11]:
```suggestion
| products | [[Product](#product-spec)] | List of `Product` offered in the application. |
```

File: product/examples/ProductCollectionPlanet.json [hunk 5]
```diff
@@ -204,6 +204,6 @@
         "title": "Next page of Products",
         "method": "GET",
         "type": "application/geo+json",
-        "href": "stat-api.example.com?page=2"
+        "href": "stapi.example.com?page=2"
```
Comment (m-mohr) [inline product/examples/ProductCollectionPlanet.json:207]:
```suggestion
        "href": "https://stapi.example.com?page=2"
```

**Review by m-mohr — CHANGES_REQUESTED** [review]

**Review by nic-planet — APPROVED** [review]

**Review by m-mohr — APPROVED** [review]

**Pull request merged**